pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -152,10 +172,22 @@ stdenv.mkDerivation rec {
     "-Donnxruntime_USE_FULL_PROTOBUF=OFF"
   ] ++ lib.optionals pythonSupport [
     "-Donnxruntime_ENABLE_PYTHON=ON"
+  ] ++ lib.optionals cudaSupport [
+    "-Donnxruntime_USE_CUDA=ON"
+    "-Donnxruntime_CUDNN_HOME=${cudaPackages.cudnn}"
+    "-DCMAKE_CUDA_ARCHITECTURES=75"
```

Comment:
`cudaPackages.cudaFlags.cudaCapabilities` is what you're looking for here. Check out [the magma derivation](https://github.com/NixOS/nixpkgs/blob/d48c7318283b64de6a41b2c68e28fc292a755ca0/pkgs/development/libraries/science/math/magma/generic.nix#L19C21-L19C61) for an example.